src/task.rs

```diff
@@ -84,3 +85,85 @@ impl ::std::fmt::Display for Task
         Ok(())
     }
 }
+
+#[cfg(test)]
```

Comment:
I already wrote some tests in `tests/task.rs`. I dislike long file mixing code & tests.